62.CAD/common.hlsl

```diff
@@ -56,20 +60,19 @@ struct ClipProjectionData
 
 struct Globals
 {
-    double4x4 viewProjection; // 128 
-    float screenToWorldRatio; // 132
-    float worldToScreenRatio; // 136
-    uint2 resolution; // 144
-    float antiAliasingFactor; // 148
-    float _pad; // 152
+    ClipProjectionData defaultClipProjection; // 88
+    double screenToWorldRatio; // 96
+    float worldToScreenRatio; // 100
+    uint32_t2 resolution; // 108
+    float antiAliasingFactor; // 112
```

Comment:
instead of annotating the offsets, slap a `#ifndef __HLSL_VERSION` and just `static_assert(offsetof())` after the declaration of the struct, that makes it much much much more rock-solid

Reply:
That's originally me btw

Reply:
I guessed

Reply:
done